libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -265,13 +266,24 @@ public Map<String, List<MitigationRoute>> getCimitConfig() throws ConfigExceptio
     }
 
     public boolean enabled(FeatureFlag featureFlag) {
-        return Boolean.parseBoolean(
-                getSsmParameter(ConfigurationVariable.FEATURE_FLAGS, featureFlag.getName()));
+        return enabled(featureFlag.getName());
     }
 
     public boolean enabled(String featureFlagValue) {
-        return Boolean.parseBoolean(
-                getSsmParameter(ConfigurationVariable.FEATURE_FLAGS, featureFlagValue));
+        boolean enabled = false;
+        try {
+            enabled =
+                    Boolean.parseBoolean(
+                            getSsmParameter(ConfigurationVariable.FEATURE_FLAGS, featureFlagValue));
+        } catch (SsmException ex) {
+            LOGGER.warn(
+                    (new StringMapMessage())
+                            .with(
+                                    LOG_MESSAGE_DESCRIPTION.getFieldName(),
+                                    "SSM parameter not found for feature flag:"
+                                            + featureFlagValue));
+        }
+        return enabled;
```

Comment:
minor thing, but I think it'd be clearer to get rid of the variable and explicitly `return false` inside the catch

Reply:
done